exif2timestream.py

```diff
@@ -10,14 +9,25 @@
 from itertools import cycle
 from inspect import isclass
 import logging
-
+import re
+import pexif
+import exifread as er
+import warnings
+SKIMAGE = False
+try:
+    from skimage.transform import rescale
+    from skimage import novice
+    import skimage.io as io
```

Comment:
I'd be keener to do `import skimage` and use `skimage.io.whatever`, as there's a [stdlib io module](https://docs.python.org/3/library/io.html#module-io)